<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>20</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Environment</string>
     </property>
     <property name="buddy">
      <cstring>projectComboBox</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="projectComboBox">
     <item>
      <property name="text">
       <string>project</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="sequenceComboBox">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <item>
      <property name="text">
       <string>seq</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="shotComboBox">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <item>
      <property name="text">
       <string>shot</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>projectComboBox</tabstop>
  <tabstop>sequenceComboBox</tabstop>
  <tabstop>shotComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
